samples/State-Management-Bot/data_models/conversation_data.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+class ConversationData:
+    def __init__(self, timestamp: str = None, channel_id: str = None, prompted_for_user_name: bool = False):
+        self._timestamp = timestamp
+        self._channel_id = channel_id
+        self._prompted_for_user_name = prompted_for_user_name
+
+    @property
```

Comment:
Since the getters and setters don't have any validation or additional logic baked in, having these are a little superfluous. It's more pythonic to not use the `@property` and `@property_name.setter` unless necessary.

As we spoke irl, there are a couple of other data model classes in the libraries that should be updated as well to just have public attributes instead of these getters and setters.